api/swagger-spec/resourceListing.json

```diff
@@ -10,6 +10,14 @@
     "description": "get available API versions"
    },
    {
+    "path": "/apis/experimental/v1alpha1",
```

Comment:
swagger-ui uses them. But that is not auto-updated, so it didnt break when we added /apis here without adding its spec. It will break, whenever we update it.